OvmfPkg/OvmfPkgX64.dsc

```diff
@@ -28,11 +28,20 @@
   # Defines for default states.  These can be changed on the command line.
   # -D FLAG=VALUE
   #
-  DEFINE SECURE_BOOT_ENABLE      = FALSE
-  DEFINE SMM_REQUIRE             = FALSE
+  DEFINE SECURE_BOOT_ENABLE      = TRUE
+  DEFINE SMM_REQUIRE             = TRUE
   DEFINE SOURCE_DEBUG_ENABLE     = FALSE
-  DEFINE TPM_ENABLE              = FALSE
-  DEFINE TPM_CONFIG_ENABLE       = FALSE
+  DEFINE TPM_ENABLE              = TRUE
+  DEFINE TPM_CONFIG_ENABLE       = TRUE
+  DEFINE SATA_PASSWORD_ENABLE    = TRUE
+  DEFINE OPAL_PASSWORD_ENABLE    = TRUE
+  DEFINE LOAD_OPTION_ROMS        = FLASE
+  DEFINE DASHARO_SYSTEM_FEATURES_ENABLE = TRUE
+  DEFINE USE_CBMEM_FOR_CONSOLE   = FALSE
+  DEFINE ABOVE_4G_MEMORY         = FALSE
+  DEFINE DISABLE_MTRR_PROGRAMMING = TRUE
+  DEFINE IOMMU_ENABLE            = FALSE

```

Comment:
These will not be used by QEMU/OVMF, please remove

Reply:
I will remove those four lines.